internal/command/user_machine_test.go

```diff
@@ -427,11 +427,12 @@ func TestCommandSide_AddMachine(t *testing.T) {
 				idGenerator:     tt.fields.idGenerator,
 				checkPermission: newMockPermissionCheckAllowed(),
 			}
+			state := domain.UserStateInitial
```

Comment:
please add the state as an argument to the tests (add at least a single new one with a deactivated state; current ones do not need to pass it)